content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Use a user namespace with a stateless pod
+reviewers:
+content_type: task
+weight: 160
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+This page shows how to configure a user namespace for stateless pods. This
+allows to isolate the user running inside the container from the one in the
```

Comment:
```suggestion
allows the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} to isolate the
operating system user identity for processes running inside the container from the user
identity as seen by the
```

Reply:
I don't link this, "the operating system user identity" seems like a very complicated way of saying user ID, that we can just say user.

Reply:
Also, is the kubelet really needed in this overview? It seems like too much detail for this (also, is the kubelet, the high-level container runtime or the low level container runtime doing this?)

Reply:
Reverted this. The details about the kubelet being involved are explained at a later section in the concept page.

Reply:
Let me know what you think